v03_pipeline/bin/pipeline_worker.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import datetime
+import os
+import time
+
+import luigi
+
+from v03_pipeline.api.model import LoadingPipelineRequest
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.paths import (
+    loading_pipeline_queue_path,
+    project_pedigree_path,
+    project_remap_path,
+)
+from v03_pipeline.lib.tasks import (
+    UpdateCachedReferenceDatasetQueries,
+    UpdateVariantAnnotationsTableWithNewSamplesTask,
+    WriteProjectFamilyTablesTask,
+)
+
+logger = get_logger(__name__)
+
+
+def main():
+    while True:
+        try:
+            if not os.path.exists(loading_pipeline_queue_path()):
+                continue
+            with open(loading_pipeline_queue_path()) as f:
+                lpr = LoadingPipelineRequest.model_validate_json(f.read())
+            project_remap_paths = [
+                project_remap_path(
+                    lpr.reference_genome,
+                    lpr.dataset_type,
+                    lpr.sample_type,
+                    project_guid,
+                )
+                for project_guid in lpr.projects_to_run
+            ]
+            project_pedigree_paths = [
+                project_pedigree_path(
+                    lpr.reference_genome,
+                    lpr.dataset_type,
+                    lpr.sample_type,
+                    project_guid,
+                )
+                for project_guid in lpr.projects_to_run
+            ]
+            task_kwargs = {
+                k: v for k, v in lpr.model_dump().items() if k != 'projects_to_run'
+            }
+            tasks = [
+                UpdateCachedReferenceDatasetQueries(
+                    reference_genome=lpr.reference_genome,
+                    dataset_type=lpr.dataset_type,
+                ),
+                UpdateVariantAnnotationsTableWithNewSamplesTask(
+                    project_guids=lpr.projects_to_run,
+                    project_remap_paths=project_remap_paths,
+                    project_pedigree_paths=project_pedigree_paths,
+                    run_id=datetime.datetime.now(datetime.timezone.utc).strftime(
+                        '%Y%m%d-%H%M%S',
+                    ),
+                    force=False,
+                    **task_kwargs,
+                ),
+                *[
+                    WriteProjectFamilyTablesTask(
+                        project_guid=lpr.projects_to_run[i],
+                        project_remap_path=project_remap_paths[i],
+                        project_pedigree_path=project_pedigree_paths[i],
+                        force=False,
+                        **task_kwargs,
+                    )
+                    for i in range(len(lpr.projects_to_run))
+                ],
+            ]
+            luigi.build(tasks)
```

Comment:
There are two options for scheduling here:
- Run with `--local-scheduler`, which means this process will handle scheduling.  This is what we're doing now in dataproc and works fine.
- We stand up a second process in the pod that's running the luigi central scheduler (literally just the `luigid` command) that will expose the luigi UI.  

My preference is to at least try to get the centralized scheduler working since it appears lightweight and the UI is passable.  It was super easy to get working locally (I went down the rabbit hole for ~an hour or so last week when trying to run tasks inside of `aiohttp` and came up pretty quickly).  The main unknowns there are how to expose the  `luigid` endpoint in `helm` correctly, but I think this is at most a day of futzing.

Reply:
Agreed that if we can get this done in under about a week it would be worth it, having a UI exposed to users so they can see how pipeline runs are progressing would be a very valuable feature